assets/client.go

```diff
@@ -184,6 +185,66 @@ func (c *TapdClient) GetAssetName(ctx context.Context,
 	return assetName, nil
 }
 
+// GetAssetPrice returns the price of an asset in satoshis. NOTE: this currently
+// uses the rfq process for the asset price. A future implementation should
+// use a price oracle to not spam a peer.
+func (c *TapdClient) GetAssetPrice(ctx context.Context, assetId string,
+	assetAmt uint64, satMinAmt btcutil.Amount) (btcutil.Amount,
```

Comment:
can't assetAmt also be of type `btcutil.Amount`, or both of type `uint64` to omit casting?